edm4eic.yaml

```diff
@@ -275,8 +282,23 @@ datatypes:
       - int32_t           sector            // The sector this hit occured in
       - edm4hep::Vector3f local             // The local position of the hit in detector coordinates (relative to the sector) [mm]
 
+  edm4eic::CherenkovParticleID:
```

Comment:
Does this need to include a set of PMT hits (tracker hits) that were used? I assume you may decide some were noise. Is there no vector member link to the hits needed?

Reply:
Good idea! I added a 1-N relation to the `MCRecoTrackerHitAssociation`s. This is good for our simulation purposes where we can know the MC truth, but in real data we won't know the truth; in the real-data case, we could still use `MCRecoTrackerHitAssociation` but the `simHits` relation would simply not be filled, so it still works. What do you think?